modules/manage/pages/tiered-storage.adoc

```diff
@@ -1,7 +1,7 @@
-= Tiered Storage
+= About Tiered Storage
 :description: Configure your Redpanda cluster to offload log segments to cloud storage and save storage costs.
 :page-context-links: [{"name": "Linux", "to": "manage:tiered-storage.adoc" },{"name": "Kubernetes", "to": "manage:kubernetes/storage/tiered-storage/k-tiered-storage.adoc" } ]
-:page-aliases: data-management:shadow-indexing.adoc, data-management:tiered-storage.adoc
+:page-aliases: data-management:shadow-indexing.adoc, data-management:tiered-storage.adoc, manage:data-archiving.adoc
 :page-categories: Management, High Availability, Data Replication
 :env-linux: true
 
```

Comment:
@kbatuigas This page is somehow missing the admonition that it needs an enterprise license. It's noted in the prereqs section, but please add the admonition to the top of the page!

Reply:
+1